gengo-v2/README.md

```diff
@@ -0,0 +1,41 @@
+# Gengo: a framework for building simple code generators
```

Comment:
Maybe also state that we might change the interface of this module in backward incompatible ways to accelerate k8s development?